src/slangPrinter.ts

```diff
@@ -82,8 +62,6 @@ function genericPrint(
   options: ParserOptions<AstNode>,
   print: PrintFunction
 ): Doc {
```

Comment:
If this PR is explicitly dropping support for v2, I wonder if this should be kept, specifying `3.0.0` instead?